the-event-loop/interval.js

```diff
@@ -0,0 +1,3 @@
+var count = 3;
+function tick(t) { var tock = setInterval(() => { if (t > 0) { console.log(t); t--; } else { clearInterval(tock); console.log('Blast off!'); } }, 1000); }
```

Comment:
It is best to break up you code on multiple line, everything on a single line is hard to read.

Reply:
sorry, i actually had it on multiple lines but it was easier for me to read this way lol. i just didnt shift it out again.